src/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -105,14 +105,16 @@ public sealed partial class EnumBuilder : System.Type //TYPEINFO: System.Reflect
         protected override bool IsArrayImpl() { throw null; }
 //TYPEINFO        public override bool IsAssignableFrom(System.Reflection.TypeInfo typeInfo) { throw null; }
         protected override bool IsByRefImpl() { throw null; }
-        public override bool IsByRefLike { get { throw null; } }
         protected override bool IsCOMObjectImpl() { throw null; }
         public override bool IsDefined(System.Type attributeType, bool inherit) { throw null; }
         protected override bool IsPointerImpl() { throw null; }
         protected override bool IsPrimitiveImpl() { throw null; }
+#if !TargetsNetstandard
+        public override bool IsByRefLike { get { throw null; } }
```

Comment:
Non-blocking comment: These could be declared in the s.r.e.netstandard.cs file rather than using #if?

Other than that, looks plausible to me though Wes will have do to review all the config/msbuild stuff.

Reply:
Yes given the number of #ifdef's you should move them to another file.

Reply:
Will do